presto-native-execution/presto_cpp/main/common/Configs.h

```diff
@@ -335,6 +349,12 @@ class SystemConfig : public ConfigBase {
   static constexpr std::string_view kExchangeEnableConnectionPool{
       "exchange.http-client.enable-connection-pool"};
 
+  /// Floating point number used in calculating how many threads we would use
+  /// for Exchange HTTP client IO executor: hw_concurrency x multiplier.
+  /// 1.0 is default.
+  static constexpr std::string_view kExchangeHttpClientNumIoThreadsHwMultiplier{
```

Comment:
I'd just call kHttpClientNumIoThreadsHwMultiplier

Reply:
> kHttpClientNumIoThreadsHwMultiplier

@xiaoxmeng 

It is used by the exchange only.